### ISSUE TITLE
No code to deal with leaving a room

### ISSUE THREAD

**Issue by jaller94** [benparsons/elizabot#1]
There is no event handling for when eliza gets kicked out of a room.
Even if this may not be required, I think it would be good to delete unneeded instances in the `elizas` array.

**Comment by benparsons** [comment]
Agree, this would be a great way of handling the list of instances.
